validation/src/main/java/uk/gov/pay/commons/validation/DateTimeUtils.java

```diff
@@ -50,4 +53,13 @@
     public static String toLocalDateString(ZonedDateTime zonedDateTime) {
         return zonedDateTime.withZoneSameInstant(ZoneOffset.UTC).toLocalDate().format(localDateFormatter);
     }
+
+    public static Optional<ZonedDateTime> fromLocalDateOnlyString(String localDateString) {
+        try {
+            ZonedDateTime utcTime = LocalDate.parse(localDateString, localDateFormatter).atStartOfDay().atZone(UTC);
```

Comment:
Since `localDateFormatter` is `DateTimeFormatter.ISO_LOCAL_DATE`, you don’t need to pass it in to `LocalDate.parse(…)` because it’s already the default:

```java
ZonedDateTime utcTime = LocalDate.parse(localDateString).atStartOfDay().atZone(UTC);
```

Also, there’s a single method that will do the job of the last two:

```java
ZonedDateTime utcTime = LocalDate.parse(localDateString).atStartOfDay(UTC);
```